alacritty_terminal/src/event_loop.rs

```diff
@@ -43,6 +43,7 @@ pub struct EventLoop<T: tty::EventedPty, U: EventListener> {
     tx: Sender<Msg>,
     terminal: Arc<FairMutex<Term<U>>>,
     event_proxy: U,
+    hold: bool,
     ref_test: bool,
```

Comment:
We're passing two config values here, usually at that point it's shorter to just pass `&config` in and then take out the specific fields in the struct construction.

Reply:
You're right, I should've thought about that. Just did.